spec/tbd_osm_suite_spec.rb

```diff
@@ -25,6 +25,19 @@ def get_clean_env
   # number of processors to use
   nproc = [1, Parallel.processor_count - 2].max
 
+  # Fetch 'OpenStudio Results' measure (if missing).
```

Comment:
NREL OpenStudio measures as required spec gems (instead of maintaining static versions within TBD). A ```bundle update``` should be sufficient to retrieve fixes by NREL, updated copyright years, etc. Needed some OSW and gemspec tweaks.